active/Expander/Expander.md

```diff
@@ -0,0 +1,155 @@
+and [Windows Community Toolkit](https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/expander).  
+
+# API Pages
+
+## Expander class
+
+Represents the control that displays a header and a collapsible content area.
+
+> Spec note: the Expander class and ExpandDirection enum names match WPF.
+
+An Expander control is a UI component that provides a standard interaction for
+showing more content in a container that pushes adjacent content while expanding and collapsing.
+An Expander cannot be light dismissed and is independent of the contents inside it, including controls.
+Expander should be used when some content is only relevant some of the time (for example to
+read more information or access additional options for an item). 
+
+![A collapsed Expander that is expanded (pushing downwards) and then collapsed. The Header has the text "This is in the header" and the Content has the text "This is in the content". There is a TextBlock saying "This TextBlock is Below the Expander" that is pushed away](images/Expander_with_textblock.gif)
+
+### Examples
+
+#### Create an Expander
+
+![A collapsed Expander that is expanded and then collapsed. The Header has the text "This is in the header" and the Content has the text "This is in the content".](images/Expander.gif)
+
+XAML
+~~~~
+<muxc:Expander x:Name="DefaultExpander" 
+    Header="This text is in the header" 
+    Content="This is in the content"/>
+~~~~
+
+#### Put controls inside an Expander
+
+![A collapsed Expander. The Header has the text "This is in the header" and the Content has a button with the text "This ToggleButton is in the Content". The Expander is expanded, the button is toggled on and off, and the Expander is collapsed. ](images/Expander_togglebutton.gif)
+
+XAML
+~~~~
+<muxc:Expander x:Name="Expander2" Header="This is in the header"> 
+    <ToggleButton>This ToggleButton is in the Content</ToggleButton>
+</muxc:Expander>
+~~~~
+
+Updated pictures with final visuals to come.
+
+### Theme Resources
+
+You can modify the look of an Expander by specifying Xaml resources in your app.
+For more info, see the
+[lightweight styling guide](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling).
+
+| Name| Description | 
+| :---------- | :------- | 
+| ExpanderChevronMargin | Chevron margin thickness| 
+| ExpanderChevronGlyph | Chevron glyph| 
+| ExpanderChevronSize | Chevron size|
+| ExpanderPopinVerticalOffset | vertical offset for animation| 
+
+Some theme resources TBD.
+
+
+
+## Expander members
+
+### Properties
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ExpandDirection | Sets the direction of expansion | Down = 0 |
+| IsExpanded | Whether or not control is expanded | False |
+
+### Events
+| Name | Description | 
+| :---------- | :------- | 
+| Expanding | Occurs when expanded |
```

Comment:
Should be: occurs when expand- _ing_. Since this is an ing event, it is raised before the content is shown, and so you could use this to delay-load the content?